### PR TITLE
SJQ2T-98: BE - Endpoint for Delete company

### PR DESCRIPTION
**Pull request by DraganMitrasinovicHtec**
Added an endpoint for deleting companies. It's a soft delete, just updating the deletedOn field to the current date.

### CODE REVIEW

File: backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java [hunk 1]
```diff
@@ -170,4 +168,71 @@ public ResponseEntity<CompanyResponse> addCompany(
...
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = String.class),
+                  examples = {@ExampleObject(value = "Internal Server Error")})
+            })
+      })
+  @PreAuthorize("hasAuthority(T(com.htecgroup.skynest.model.entity.RoleEntity).ROLE_ADMIN)")
+  @DeleteMapping("/{uuid}")
+  @ResponseStatus(value = HttpStatus.OK)
+  public void deleteCompany(@PathVariable UUID uuid) {
```
Comment (ivanfhtec) [inline backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java:234]: void?

Comment (DraganMitrasinovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java:234]: Do we really need to return string for delete operations? You can tell if it's successful by the status code?

Comment (MetoMladenovski) [reply backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java:234]: ResponseEntity(Boolean) i think is the best

Comment (ivanfhtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java:234]: either one is fine, as long as we're consistent

Comment (DraganMitrasinovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/CompanyController.java:234]: No prob, will change

**Pull request closed without merging**